<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ctkRangeWidget</class>
 <widget class="QWidget" name="ctkRangeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>302</width>
    <height>27</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>ctkSliderSpinBoxWidget</string>
  </property>
  <layout class="QGridLayout" name="GridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="ctkSpinBox" name="MinimumSpinBox"/>
   </item>
   <item row="0" column="1">
    <widget class="ctkDoubleRangeSlider" name="Slider">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="ctkSpinBox" name="MaximumSpinBox"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkDoubleRangeSlider</class>
   <extends>QWidget</extends>
   <header>ctkDoubleRangeSlider.h</header>
  </customwidget>
  <customwidget>
   <class>ctkSpinBox</class>
   <extends>QWidget</extends>
   <header>ctkSpinBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
